cmd/minikube/cmd/delete.go

```diff
@@ -723,3 +736,40 @@ func getPids(path string) ([]int, error) {
 
 	return pids, nil
 }
+
+// deleKnownHost read minikube nodes' keys from the minikube home folder
+// and remove them from the known_hosts file
+// This is the long term solution for issue https://github.com/kubernetes/minikube/issues/16868
```

Comment:
Hi, just a random question from a random person

How does this PR contribute to fixing https://github.com/kubernetes/minikube/issues/16868 ?

I see that issue stopped happening since August ([here](https://storage.googleapis.com/minikube-flake-rate/flake_chart.html?env=Docker_Linux_containerd&test=TestDockerEnvContainerd&period=last90))

Sorry if the question is too basic, I'm just learning

Reply:
The test was fixed with https://github.com/kubernetes/minikube/pull/16935 because that PR clears out the known_hosts file after finishing testing in our testing infra. But real users are still experiencing the issue

Reply:
Oh, I got it now! thanks